### PR TITLE
gctcli: bump urfave cli depends version

### PR DESCRIPTION
**Pull request by shazbert**
# PR Description

Had a stale branch I found locally, opened if we want to merge. Seems like we were already ref: v2. 🤷 

Fixes # (issue)

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [ ] go test ./... -race
- [ ] golangci-lint run
- [ ] Test X

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: cmd/gctcli/commands.go [hunk 1]
```diff
@@ -3521,31 +3521,31 @@ func getExchangeTickerStream(c *cli.Context) error {
...
-var getAuditEventCommand = cli.Command{
+var getAuditEventCommand = &cli.Command{
 	Name:      "getauditevent",
 	Usage:     "gets audit events matching query parameters",
 	ArgsUsage: "<starttime> <endtime> <orderby> <limit>",
 	Action:    getAuditEvent,
 	Flags: []cli.Flag{
-		cli.StringFlag{
+		&cli.StringFlag{
 			Name:        "start, s",
```
Comment (thrasher-) [inline cmd/gctcli/commands.go:3531]: These will need to be adjusted as per https://github.com/urfave/cli/blob/master/docs/migrate-v1-to-v2.md#flag-aliases-are-done-differently

**Review by thrasher- — CHANGES_REQUESTED** [review]
Just 1 nit plus addidtional request:

Could you please also add in the latest autocomplete templates as per https://github.com/urfave/cli/tree/v2.3.0/autocomplete since they contain some bug fixes

**Review by gloriousCode — COMMENTED** [review]
I can confirm that it all works, including aliases:
```
.\gctcli.exe trade getrecent -e=binance -pair=ETH-BTC -a=spot
{
 "exchange_name": "binance",
 "asset": "spot",
 "pair": {
  "delimiter": "-",
  "base": "ETH",
  "quote": "BTC"
 },
 "trades": [
  {
   "price": 0.057707,
   "amount": 1.6,
   "timestamp": "2021-06-22 23:56:37 UTC",
   "trade_id": "278791697"
  },
...
```

I'll check out the autocomplete once complete

**Review by gloriousCode — APPROVED** [review]
tACK!
image
image
image

**Review by MadCozBadd — APPROVED** [review]
Tested and Approved! Nice changes

**Review by thrasher- — APPROVED** [review]
Thank you for addressing the nits!

**Pull request merged**